plugin-agent-admin-v2-TS/src/components/AgentAdminView/AgentAdminView.tsx

```diff
@@ -76,9 +80,11 @@ const AgentAdminView = () => {
   }
 
   return (
-      <Box overflow='auto' maxHeight='100%'>
-        <Flex>
-          <Table>
+    <ScrollWrapper>
```

Comment:
I'm not a fan of using the styled react component for something as trivial as width of a component. We can jump on a call and get it configured together, but introducing styled for this seems unnecessary and it complicates the code.